### PR TITLE
Fix#5672

### PR DESCRIPTION
**Pull request by tsarangi**
Read latest condor log. Read only non-empty condor logs

### CODE REVIEW

**Comment by tsarangi** [comment]
@lucacopa is still testing ... wait for his reply

**Comment by lucacopa** [comment]
@tsarangi  patch is working fine, @ticoann please merge this

**Comment by tsarangi** [comment]
@lucacopa, @ticoann 
I would suggest merger issue #5659 first and then merge #5672. This way the conflicts won't be there...

**Comment by ticoann** [comment]
Thanks, I will merge in that order.

**Comment by lucacopa** [comment]
@ticoann you can merge this right away. #5679 needs more work, I talked with @tsarangi about that

**Comment by ticoann** [comment]
@tsarangi, Hi Tapas, I merged #5659 first but still has a conflict for this merge. Could you check on that and update patch, sorry.

**Comment by tsarangi** [comment]
@ticoann  Hi Seangchan,
It might be easier to delete this pull request and I will create another pull request with a clean branch...

**Comment by tsarangi** [comment]
@ticoann 
May be wait. I think I solved the conflict. I will push in the changes and rebase into the same branch

**Comment by ticoann** [comment]
Thanks Tapas

**Comment by tsarangi** [comment]
@ticoann  Can you try and merge this now ! it should work.

File: src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py [hunk 1]
```diff
@@ -1011,9 +1011,6 @@ def makeSubmit(self, jobList):
 
             jdl.append("priority = %i\n" % (task_priority + prio*self.maxTaskPriority))
 
-            jdl.append("+PostJobPrio1 = -%d\n" % len(job.get('potentialSites', [])))
```
Comment (lucacopa) [inline src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py]: Is this really wanted? I think this lines were added by the priority patch

File: src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py [hunk 2]
```diff
@@ -1060,8 +1057,8 @@ def customizePerJob(self, job):
         if job.get('taskName', None):
             jdl.append('+WMAgent_SubTaskName = "%s"\n' % job['taskName'])
 
-        if job.get('taskType', None):
-            jdl.append('+CMS_JobType = "%s"\n' % job['taskType'])
+        if job.get('subTaskType', None):
```
Comment (lucacopa) [inline src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py]: Same with this one, it was pushed recently by:
https://github.com/dmwm/WMCore/commit/a42abf7638b87b064bbae9e6dcde1d9c48a8a49b#diff-9b070481bcf1072545075872a49c7968

Comment (ericvaandering) [inline src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py]: Good catches, Luis. Yeah, please stop trying to undo my work. :-)

**Comment by tsarangi** [comment]
it wasn't intentional. Anyways, I will cancel the pull request and remake one.

**Pull request closed without merging**